examples/hcp-vm-demo/output.tf

```diff
@@ -17,5 +17,14 @@ output "hashicups_url" {
 }
 
 output "next_steps" {
-  value = "Hashicups Application will be ready in ~2 minutes. Use 'terraform output consul_root_token' to retrieve the root token."
+  value = "Hashicups Application will be ready in ~5 minutes. Use 'terraform output consul_root_token' to retrieve the root token."
```

Comment:
Maybe we provide instructions here or a link to instructions on github?

Reply:
SSH instructions and/or interacting with Nomad instructions? I put instructions in the README... but I think I realize the point now as writing this that they TF UI users won't be going right to that: `examples/hcp-vm-demo/README.md`